src/helpers/augmentedReality/storageNameCreator.js

```diff
@@ -1,6 +1,13 @@
+import { documentDirectory } from 'expo-file-system';
+
 export const storageNameCreator = ({ dataItem, objectItem }) => {
-  const { id: objectId } = dataItem;
-  const { title, type, id } = objectItem;
+  const { id: objectId, title: objectTitle } = dataItem;
+  const { title, type } = objectItem;
```

Comment:
this confuses me when reading.
why does the `dataItem.title` be renamed to `objectTitle` and the `objectItem.title` becomes `title`?

Reply:
what about not destructing at all and use `dataItem.id`, `dataItem.title`, `objectItem.title` and `objectItem.type` where necessary?

Reply:
https://github.com/ikuseiGmbH/smart-village-app-app/pull/569/commits/1dbcc5eb3ef7c049a142104e92b97fe6a1ba2bda